pkgs/tools/video/vcsi/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, python3Packages, fetchFromGitHub, makeWrapper, ffmpeg }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "vcsi";
+  version = "7.0.12";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "0dks0yr2a0cpr32vrwhdrhsb4qyj7rz1yv44fjbr8z8j8p84yjx5";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  propagatedBuildInputs = with python3Packages; [
+    numpy
+    pillow
+    jinja2
+    texttable
+    parsedatetime
+    nose
+  ];
+
+  postInstall = with stdenv.lib; ''
+    wrapProgram $out/bin/vcsi \
+      --prefix PATH : ${makeBinPath [ ffmpeg ]}
+  '';
```

prevent double wrapping with wrapPython
```suggestion
  makeWrapperArgs = [ "--prefix PATH : ${lib.makeBinPath [ ffmpeg ]}" ];
```

---

I also removed wrapProgram from nativeBuildInputs